lib/rules/web/admin_console/4.4/goto.xyaml

```diff
@@ -222,4 +222,6 @@ goto_all_machines_page:
 goto_all_machine_sets_page: 
   url: k8s/all-namespaces/machine.openshift.io~v1beta1~MachineSet
 
-
+# global config
+goto_alertmanagerconfig_page:
+  url: monitoring/alertmanagerconfig
```

Comment:
Highly suggest  to make sure each `goto` works by the rule itself. Needs a `action wait_box_loaded` here. Sometimes it needs a long time out to loading page and next action may fail during loading. 
Generally I suggest the logical order : `first do some operation -> then check` in one rule, and all same (`first do some operation -> then check`) in other rule. 
I mean each single rule should follow operate -> check order. So that `Then the step should succeed` make sense.
Refer to [this PR](https://github.com/openshift/verification-tests/pull/726) opened several days ago.
HDYT?